src/docs/index.js

```diff
@@ -67,49 +68,33 @@ const FormContaner = styled.form`
   padding: 10px;
 `;
 
+const Logger = () => <Logs data={data}
+                         path="path>path?path"
+                         onClear={() => console.log('Clear all')}
+                         onResend={(item) => console.log('Resend', item)}/>
+
 const Form = () => <FormContaner>
   <Input onChange={onChange} placeholder="Default Input"/>
   <Dropdown title="Dropdown" items={items} onChange={onChange}/>
   <CheckBox label="Checkbox" onChange={onChange}/>
   <Button theme="active" text="Submit"/>
 </FormContaner>;
 
-const layout = {
-  id: guid(),
-  type: 'container',
-  orientation: 'vertical',
-  elements: [
-    {
-      id: guid(),
-      type: 'element',
-      value: 'Just text'
-    },
-    {
-      id: guid(),
-      type: 'container',
-      orientation: 'horizontal',
-      elements: [
-        {
-          id: guid(),
-          type: 'element',
-          value: 'Just text'
-        },
-        {
-          id: guid(),
-          type: 'element',
-          value: 'Just text'
-        }
-      ]
-    }
-  ]
+const creators = {
+  text: { name: 'Paragraph', element: () => <Paragraph fontSize="2.5rem">Hello, World!</Paragraph> },
+  form: { name: 'Form', element: () => <Form/> },
+  logs: { name: 'Logs', element: () => <Logger/> }
 };
 
 const App = () => (
   <Container>
     <Paragraph fontSize="3rem" color="#3F3F3F" backgroundColor="#FAFAFA">CapsulaJS UI components</Paragraph>
```

Comment:
suggest to put #FAFAFA in variable, cause it's widely used

Reply:
done